crates/re_types_builder/src/codegen/python/mod.rs

```diff
@@ -633,6 +633,20 @@ fn code_for_struct(
         code.push_indented(1, quote_init_method(reporter, obj, ext_class, objects), 2);
     }
 
+    // Generate __bool__ operator if this is a single field struct with a bool field.
+    if fields.len() == 1 && fields[0].typ == Type::Bool {
```

Comment:
I don't know if we have to restrict this to bool.

Definitely not as critical as the bool case, but for types that contain an int, for example, it would be nice to have 0 act Falsey as well.

Reply:
Hmmmm 🤔
Another time maybe